examples/basic_cog.py

```diff
@@ -77,8 +96,6 @@ async def on_member_join(self, member):
             await guild.system_channel.send(to_send)
 
 
-# This method allows us to add our Cog to the main bot. 
-# Without this your  Cog will not be added to `Bot.cogs` and therefore its listeners and commands will not be registered.
-# You can use bot.remove_cog to remove a Cog from a bot.
+# Cogs have to be explicitly added.
```

Comment:
Would adding a note about extensions be good here?

Reply:
There would be a separate example for extensions so I feel like I should keep this separate.

Reply:
well it could just be like "cogs are often used with extensions, you can read about them here: https://discordpy.readthedocs.io/en/latest/ext/commands/extensions.html#ext-commands-extensions which might be good to prompt them to looking at it, but I don't know if that's something that's good for an example.